https://gitlab.freedesktop.org/pipewire/wireplumber/-/commit/703c35cfd44265e81595db29eed081c8785cda87

From 703c35cfd44265e81595db29eed081c8785cda87 Mon Sep 17 00:00:00 2001
From: Wim Taymans <wtaymans@redhat.com>
Date: Mon, 29 Nov 2021 09:25:18 +0100
Subject: [PATCH] access-config: add restricted access permissions

When a pulse client connects with TCP, it is given the 'restricted'
access policy. Assign 'rx' permissions for those clients instead of
blocking forever.

See pipewire#1863
---
 src/config/main.lua.d/50-default-access-config.lua | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/src/config/main.lua.d/50-default-access-config.lua b/src/config/main.lua.d/50-default-access-config.lua
index 0282d4aa..6cf18bed 100644
--- a/src/config/main.lua.d/50-default-access-config.lua
+++ b/src/config/main.lua.d/50-default-access-config.lua
@@ -20,4 +20,12 @@ default_access.rules = {
     },
     default_permissions = "rx",
   },
+  {
+    matches = {
+      {
+        { "pipewire.access", "=", "restricted" },
+      },
+    },
+    default_permissions = "rx",
+  },
 }
-- 
GitLab

